Assets/VRTK/Scripts/VRTK_HeightAdjustTeleport.cs

```diff
@@ -152,7 +159,7 @@ private void DropToNearestFloor(bool withBlink, bool useGravityFall)
                         Blink(blinkTransitionSpeed);
```

Comment:
Should this be done inside the `FloorHeightChanged` conditional?

What's the point in blinking if you're not going to change position?